src/main/java/fi/otavanopisto/kuntaapi/server/discover/EntityUpdater.java

```diff
@@ -148,16 +141,23 @@ public boolean isStopped() {
     return stopped;
   }
   
-  private boolean isEligibleToRun() {
-    if (systemSettingController.inFailsafeMode()) {
-      return false;
+  private UserTransaction lookup() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this unused private "lookup" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)